### PR TITLE
Update pricing for Fall 2024

### PR DESCRIPTION
**Pull request by nwalters512**

### CODE REVIEW

File: src/pages/pricing/index.tsx [hunk 1]
```diff
@@ -187,46 +207,38 @@ export default function Pricing() {
...
-                  onClick={() => updatePaymentModel("course")}
-                >
-                  <span
-                    className={classnames("me-2", {
-                      "d-none": paymentModel !== "course",
-                    })}
+            <Row>
+              <Col xs={12} md={6}>
+                <Form.Group controlId="term-system">
+                  <Form.Label>Term system</Form.Label>
```
Comment (nwalters512) [inline src/pages/pricing/index.tsx:213]: Is this the right term (heh) to use here?

File: src/pages/pricing/index.tsx [hunk 2]
```diff
@@ -136,16 +134,38 @@ function ContactUsButton({ className }: { className?: string }) {
...
+      return 2;
+    }
+  });
+
+  let premiumPrice = basicPrice * 2;
+
+  if (paymentModel === "student") {
+    basicPrice += 2;
+    premiumPrice += 2;
+  }
```
Comment (nwalters512) [inline src/pages/pricing/index.tsx:160]: Check my math please!

File: src/pages/pricing/index.tsx (hunk 1 above)
Comment (mfsilva22) [reply src/pages/pricing/index.tsx:213]: "Term system" is a correct terminology. We could also consider "academic schedule"

File: src/pages/pricing/index.tsx (hunk 2 above)
Comment (mfsilva22) [reply src/pages/pricing/index.tsx:160]: It looks good to me!

Comment (mwest1066) [reply src/pages/pricing/index.tsx:160]: Likewise!

File: src/pages/pricing/index.tsx (hunk 1 above)
Comment (mwest1066) [reply src/pages/pricing/index.tsx:213]: I prefer "academic schedule", but I agree that "term system" is correct as well.

**Review by mwest1066 — APPROVED** [review]

File: src/pages/pricing/index.tsx (hunk 1 above)
Comment (nwalters512) [reply src/pages/pricing/index.tsx]: Changed to "academic calendar" in 7129fb5.

Comment (nwalters512) [reply src/pages/pricing/index.tsx]: err, no I didn't 😅 let me change it again.

Comment (nwalters512) [reply src/pages/pricing/index.tsx]: Done for real in 56ec012.

**Pull request merged**